<launch>

    <node name="joy_node" pkg="joy" type="joy_node">
        <param name="dev" value="/dev/input/js0"/>
    </node>

    <node pkg="joy_mux" type="joy_mux.py" name="joy_mux_node" output="screen" required="true">
        <param name="source_topic" value="/joy"/>
        <param name="button_index" value="6"/>
        <rosparam param="published_topics">[/joy/robot, /joy/arm]</rosparam>
    </node>
</launch>